sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj

```diff
@@ -9,7 +9,8 @@
 		<![CDATA[
 			Documentation can be found at https://go.microsoft.com/fwlink/p/?linkid=862452.
 			
-			Bug fixes and refactoring
+			Updating ADAL to 5.x
```

Comment:
Should the version of the package be bumped here?

Reply:
It is? Please clarify?

Reply:
I was referring to bumping the version of the 'Microsoft.Azure.Services.AppAuthentication' package

Reply:
Oh sorry! That was actually done previously by another contributor so it wasn't necessary (here is specific [commit](https://github.com/Azure/azure-sdk-for-net/commit/a8607eff533f14218bf0db13d9a24023915e3075#diff-0a0f3a64c5b86613b27100f716d8a860)).

Reply:
Ah! got it! Ok!